### DISCUSSION TITLE
No Rewards etc.

### DISCUSSION THREAD

**Discussion by Admin-Miner** [Chia-Network/chia-blockchain#7377] [Support]
I am farming since 5 days no single reward or block until now.
What means testnet ? Using Windows 10. Other coins on chianet work i am farming i get rewards ? Please explain that to me. Is that a not working blockchain here, wich is still in testing phase ?

**Comment by BrandtH22** [comment]
Since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
